exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -283,20 +308,114 @@
                                    alert(errorThrown);
                                },
                                success: function(data) {
-			           alert(data["response"]);
+                                   alert(data["response"]);
                                    button.prop('disabled',true).css('opacity',0.5);
                                }
                   });
               }
           }
-      </#if>
+
+          function remoteShutdown(button,host) {
+              var url = location.protocol + "//" + host + "/gracefulShutdown";
+              var result = $.ajax({
+                    type: 'POST',
+                    url: url,
+                    contentType : 'text/plain',
+                    complete: function(data) {
+                        alert(data.responseJSON["response"]);
+                        button.prop('disabled',true).css('opacity',0.5);
+                    }
+              });
+          }
+          </#if>
+
+      function popOutRemoteDbitUI(dbitHost, dbitPort) {
```

Comment:
this and functions below should be under `<#if !model.isAuthEnabled()> <#/if>` block

Reply:
We can't put this and the remaining functions within the IF block since the localhost operations are still valid. i.e. an authenticated client on a specific Drillbit can ping that Drillbit for refresh of the metrics. Only shutdown function needs to be managed with the IF block